Prox/Prox.xcodeproj/project.pbxproj

```diff
@@ -488,7 +503,7 @@
 				TargetAttributes = {
 					7B29E7BE1DA3EC360099AF38 = {
 						CreatedOnToolsVersion = 8.0;
-						DevelopmentTeam = 9G8J6YA743;
```

Comment:
Development team changed.

Did you know you can add partial files to the staging area with `git add -p <file-name>`? I'm not sure why the dev team changes but you could selective add the project file if this is a common problem.